config/initializers/rack_timeout.rb

```diff
@@ -2,20 +2,21 @@
 
 module Rack
   class Timeout
-    @excludes = [
-      '/api/verify/images',
-      '/verify/doc_auth/document_capture',
-      '/verify/doc_auth/verify',
-      '/verify/capture_doc/document_capture',
-      '/verify/doc_auth/link_sent',
-    ]
+    EXCLUDES = [
+      '/verify/verify_info',
+      '/verify/phone',
+      '/verify/document_capture',
+      '/verify/link_sent',
+    ].flat_map do |path|
+      [path] + Idp::Constants::AVAILABLE_LOCALES.map { |locale| "/#{locale}#{path}" }
```

Comment:
might need one more slash in there
```suggestion
      [path] + Idp::Constants::AVAILABLE_LOCALES.map { |locale| "/#{locale}/#{path}" }
```

Reply:
`path` includes a leading slash. Output of current constant:

```
[1] pry(main)> puts Rack::Timeout::EXCLUDES
/verify/verify_info
/en/verify/verify_info
/es/verify/verify_info
/fr/verify/verify_info
/verify/phone
/en/verify/phone
/es/verify/phone
/fr/verify/phone
/verify/document_capture
/en/verify/document_capture
/es/verify/document_capture
/fr/verify/document_capture
/verify/link_sent
/en/verify/link_sent
/es/verify/link_sent
/fr/verify/link_sent
/api/verify/images
```

Reply:
ohh right right. thx!